lms/views/lti/basic_launch.py

```diff
@@ -61,11 +60,13 @@ def __init__(self, context, request):
     def configured_launch(self):
         """Display a document if we can resolve one to show."""
 
-        return self._show_document(
+        self._show_document(
             document_url=self.request.find_service(DocumentURLService).get_document_url(
                 self.request
             )
         )
+        self.request.registry.notify(LTILaunchEvent(self.request))
```

Comment:
On the event type:

I think LTILaunch is a very broad concept. Deep Linking is also a type of launch but we don't merge them together for example.

Now that we can be more granular I think it makes sense to keep track of:

- Configured launches (ie a user sees a document)
- Deep linking (ie a teacher configures a document in Canvas)
- Configure assignment? (ie a teacher configures an assignment in non-canvas?) Not in this PR.